docs/develop/user-input.md

```diff
@@ -12,7 +12,7 @@ Adding parameters to your script lets other users provide data for the script, w
 :::image type="content" source="../images/user-input-example.png" alt-text="The dialog box shown to users when a script with parameters is run.":::
 
 > [!IMPORTANT]
-> Currently, users will only be prompted to enter data for parameterized scripts in Excel on the web. Power Automate flows also support giving data to scripts through parameters.
+> Currently, users will only be prompted to enter data for parameterized scripts for select preview users of Excel on the web. Power Automate flows also support giving data to scripts through parameters.
```

Comment:
"users" twice in the first sentence seems weird. Maybe this? 
```suggestion
> Currently, only a select preview of users will be prompted to enter data for parameterized scripts in Excel on the web. Power Automate flows also support giving data to scripts through parameters.
```

Reply:
Good call. I'll go with "> Currently, only select users in preview will be prompted to enter data for parameterized scripts in Excel on the web. Power Automate flows also support giving data to scripts through parameters." That'll indicate folks need to be in preview channels.